src/norm/rowutils.nim

```diff
@@ -2,11 +2,11 @@ import strutils, sequtils
 import sugar
 import macros; export macros
 import typetraits
-import oids, times
+import oids, times, options
 
 import objutils, pragmas, universal
 
-type Row = seq[string]
+type Row = seq[Option[string]]

```

Comment:
@JohnAD not really sure this is a good idea... We're fundamentally changing the way Rows are treated, breaking compat with db_* modules, rewriting all code that handles Rows, all for the _optional_ feature of null checks...

However, as a temporary hack it's OK as far as it helps you move forward. I'd love to see the complete PR, maybe it'll all make sense in the right context.

Reply:
I will be curious as to how they fix the db_* libraries. Since `nil` is being deprecated they need to find another way to handle nulls. For now, this will work, sort of, for MongoDB dev.

Whatever they come up with, I agree it's best to match their new null-handling technique.